test/float8/test_base.py

```diff
@@ -334,10 +315,6 @@ def test_autocast_outputs(
     @pytest.mark.parametrize("emulate", [True, False] if is_cuda_8_9 else [True])
     @unittest.skipIf(not torch.cuda.is_available(), "CUDA not available")
     def test_type_cast(self, linear_dtype: torch.dtype, emulate: bool):
-        emulate = (
```

Comment:
the parametrization and skipping if cuda not available should already accomplish this